ph5/clients/ph5tostationxml.py

```diff
@@ -834,7 +840,7 @@ def run_ph5_to_stationxml(paths, nickname, out_format,
                                         networks=networks,
                                         source="PIC-PH5",
                                         sender="IRIS-PASSCAL-DMC-PH5",
-                                        created=datetime.datetime.now(),
+                                        created=datetime.now(),

```

Comment:
This is using local time and should be UTC. We need to fix this without adding too much overhead.